go.mod

```diff
@@ -111,9 +112,10 @@ require (
 	gopkg.in/gcfg.v1 v1.2.0
 	gopkg.in/square/go-jose.v2 v2.2.2
 	gopkg.in/yaml.v2 v2.2.8
+	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
 	k8s.io/api v0.0.0
 	k8s.io/apiextensions-apiserver v0.0.0
-	k8s.io/apimachinery v0.0.0
+	k8s.io/apimachinery v0.19.2
```

Comment:
this looks wrong, it should be the version symlinked from staging/, we override this with a `replace`.

use `hack/update-vendor.sh` when modifying deps.

Reply:
the go.mod / vendor changes in this PR are not needed.